graphics/fontforge/Portfile

```diff
@@ -65,39 +67,34 @@ platform darwin 8 {
     configure.env-append PATH=${prefix}/libexec/gnubin/:$env(PATH)
 }
 
-configure.args      --mandir=${prefix}/share/man \
-                    --disable-python-scripting \
-                    --disable-python-extension \
-                    --without-x \
-                    --enable-woff2 \
-                    --disable-silent-rules
+configure.args-append -DENABLE_PYTHON_SCRIPTING=False \
+                      -DENABLE_PYTHON_EXTENSION=False \
+                      -DENABLE_GUI=False \
+                      -DENABLE_WOFF2=True
 
 variant python37 conflicts python38 description {Enable Python support (Python 3.7)} {
     depends_lib-append      port:python37
-    configure.args-delete   --disable-python-scripting \
-                            --disable-python-extension
-    configure.python        ${prefix}/bin/python3.7
-    configure.args-append   am_cv_python_pythondir=${frameworks_dir}/Python.framework/Versions/3.7/lib/python3.7/site-packages \
-                            am_cv_python_pyexecdir=${frameworks_dir}/Python.framework/Versions/3.7/lib/python3.7/site-packages
+    configure.args-delete   -DENABLE_PYTHON_SCRIPTING=False \
+                            -DENABLE_PYTHON_EXTENSION=False
+    configure.args-append   -DPython3_EXECUTABLE="${prefix}/bin/python3.7" \
+                            -DPYHOOK_INSTALL_DIR="${frameworks_dir}/Python.framework/Versions/3.7/lib/python3.7/site-packages"
     configure.pkg_config_path \
                             "${frameworks_dir}/Python.framework/Versions/3.7/lib/pkgconfig"
 }
 
 variant python38 conflicts python37 description {Enable Python support (Python 3.8)} {
     depends_lib-append      port:python38
-    configure.args-delete   --disable-python-scripting \
-                            --disable-python-extension
-    configure.python        ${prefix}/bin/python3.8
-    configure.args-append   am_cv_python_pythondir=${frameworks_dir}/Python.framework/Versions/3.8/lib/python3.8/site-packages \
-                            am_cv_python_pyexecdir=${frameworks_dir}/Python.framework/Versions/3.8/lib/python3.8/site-packages
+    configure.args-delete   -DENABLE_PYTHON_SCRIPTING=False \
+                            -DENABLE_PYTHON_EXTENSION=False
+    configure.args-append   -DPython3_EXECUTABLE="${prefix}/bin/python3.8" \
+                            -DPYHOOK_INSTALL_DIR="${frameworks_dir}/Python.framework/Versions/3.8/lib/python3.8/site-packages"
     configure.pkg_config_path \
                             "${frameworks_dir}/Python.framework/Versions/3.8/lib/pkgconfig"
 }
 
 variant gui description {Enable GUI support} {
-    depends_lib-append      port:xorg-libXi port:gtk3
-    configure.args-replace  --without-x --with-x
-    configure.args-append   --enable-gdk=yes
+    depends_lib-append      port:gtk3
```

Comment:
Does this port generate `FontForge.app` inside Applications?
It would be nice if it did, see how it was done in #6289.

Reply:
The CMake build has a `macbundle` target that makes `FontForge.app` ([here](https://github.com/fontforge/fontforge/blob/0686c0609632cbe876e8038871d70542108a8613/osx/CMakeLists.txt)). It appears somewhat more complete than the one in #6289. For example, it has a more complete `Info.plist` that has file associations and icons. Problem is, that target doesn’t work at all (at least not without `realpath`, so `coreutils` would be a build-time dependency).

I’d prefer to use that more complete `FontForge.app` instead, but I haven’t had a chance to work through it yet. I don’t personally think it should block this update, since the existing port is broken, and this replaces it with a current and working version.

Reply:
I took a closer look at upstream’s `macbundle` target and it’s not so hot either.

So, in that it’s still better than nothing and not really any worse than the alternative, I borrowed your approach from #6289 to build a small shim app in the +gui variant.

This should be ready to merge now. Thanks!